packages/genesys-spark-components/src/components/stable/gux-listbox-multi/gux-listbox-multi.tsx

```diff
@@ -257,8 +257,10 @@ export class GuxListboxMulti {
         listboxOption.value
       );
       if (this.filterType !== 'custom') {
-        listboxOption.filtered = !listboxOption.textContent
-          .trim()
+        listboxOption.filtered = !listboxOption.shadowRoot
+          .querySelector('slot')
+          ?.assignedNodes()[0]
+          ?.textContent.trim()
```

Comment:
This allows the user to slot things in any order. For some reason, that querySelector plus assignNodes only grabs the default slot and not the unnamed ones